Gemfile

```
@@ -26,11 +26,13 @@ gem 'puma', '~> 3.7'
 gem 'active_model_serializers', '~> 0.10.0'
 gem 'devise_token_auth'
 gem 'rack-reducer', require: 'rack/reducer'
+gem 'sidekiq'
```

Remember always tag gem version [Check it](https://guides.rubygems.org/patterns/#pessimistic-version-constraint)
```suggestion
gem 'sidekiq', '~>5.2.7'
```